awscli/examples/devicefarm/list-devices.rst

```diff
@@ -0,0 +1,27 @@
+### To list all iOS private devices and their UDID value
```

Comment:
Would you be able to format and phrase this file similar to how other rst example files are formatted? DynamoDB's `create-table.rst` example file could be a good reference: https://raw.githubusercontent.com/aws/aws-cli/develop/awscli/examples/dynamodb/create-table.rst